test/cbmc/proofs/Cellular_ATRemovePrefix/Cellular_ATRemovePrefix_harness.c

```diff
@@ -54,7 +54,8 @@ void harness()
 {
     uint32_t stringLength;
 
-    __CPROVER_assume( stringLength < CELLULAR_AT_MAX_STRING_SIZE + 1 );
+    /* Assume the size is just longer than CELLULAR_AT_MAX_STRING_SIZE + "\0". */
+    __CPROVER_assume( stringLength <= CELLULAR_AT_MAX_STRING_SIZE + 2 );
```

Comment:
The reason using __CPROVER_assume is because that `strnlen` can restrict the maximum buffer size. Once changing to `strlen`, `stringLength < CELLULAR_AT_MAX_STRING_SIZE + 1` will be no longer meaningful. We may need to consider the real buffer size to decide this length.

Reply:
Use other implementation.